web-api/nishiki-web-api.yaml

```diff
@@ -194,6 +201,7 @@ paths:
                   type: string
                   nullable: false
```

Comment:
I think the food name can be null when you update a food.

Reply:
Please let me check your point out.
Do you mean that you think we should create endpoints for each element of the food separately? I don't think so; this endpoint accepts only the full set of elements, even if only one element is changed.

Reply:
@hitohata 
I'm sorry for the late review. Thank you for your explanation. I got your point. Now, we have changed our specifications so that unit and quantity are no longer required properties. But I approve this PR now to allow you to make these change on another branch.